CHANGELOG

```diff
@@ -1,6 +1,8 @@
 devel
 -----
 
+* Added read write metrics
+
```

Comment:
The convention for our CHANGELOG is to end every entry with a full stop or something equivalent.

In addition, "Added read-write metrics" is not very understandable for end users or other interested parties that browse our CHANGELOG. What about

```
* Added metrics for document read and write operations:

  - `arangodb_num_document_reads`: Total number of document reads since process start.
  - `arangodb_num_document_writes: Total number of document writes since process start
  - `arangodb_num_document_replications`: Number of document replications since process start
  ...
```